holidays/countries/solomon_islands.py

```
@@ -0,0 +1,209 @@
+    SAT_SUN_TO_NEXT_MON_TUE,
+    SUN_TO_NEXT_MON,
+)
+
+
+class SolomonIslands(
+    ObservedHolidayBase, ChristianHolidays, InternationalHolidays, StaticHolidays
+):
+    """Solomon Islands holidays.
+
+    References:
+        * [Public Holidays Act](https://web.archive.org/web/20250615054807/https://www.paclii.org/sb/legis/consol_act_1996/pha163.pdf)
```

Pylint (3.3.7)

[convention] Line too long (134/100)

(C0301)